airbyte-webapp/src/hooks/services/useConnector.tsx

```diff
@@ -51,6 +34,18 @@ const useConnector = (): ConnectorService => {
     );
   };
 
+  return { updateAllSourceVersions };
+};
+
+export const useUpdateDestinationDefinitions = () => {
+  const { destinationDefinitions } = useDestinationDefinitionList();
+  const { mutateAsync: updateDestinationDefinition } = useUpdateDestinationDefinition();
+
+  const newDestinationDefinitions = useMemo(
+    () => destinationDefinitions.filter(Connector.hasNewerVersion),
+    [destinationDefinitions]
+  );
+
```

Comment:
These two methods were just moved into their own hooks so definitions aren't loaded for the count any more.